structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,135 @@
+3. Once the Collaborators informally agree that everyone has had enough time to read the message and ask questions, the vote can be created, with the option to approve or deny, according to the process defined in this document. No veto can be used in this vote.
+4. After the poll is closed, the vote is considered passed if more than 50% of the votes cast are positive. The discussion channel must be archived and kept read-only.
+- If the vote has passed: the community is removed from the program and from any listing that may exist.
+- If the vote is rejected: the community is reintegrated to the program, and roles are given back if necessary. Internal discussions can be purged first if needs be.
+
+# Rationale and alternatives
+
+The Quilt Community Collab program existed prior to this document. This document tries to summarize how the server has been working so far while formalising some aspects that were still left up in the air.
+
+More importantly, three polls have been held inside the server, with the following results:
+
+- When voting for changes on who is or isn't part of the CC network, how should we count votes?
+    - 10 answered: One vote per community: each community will have to write down a formal process on which they decide to vote for or against changes. This process must be democratic and fair.
+    - 4 answered: One vote per collaborator: each user or plural system present in the server gets one vote
+- When voting on who is or isn't part of the CC network, what proportions of approvals versus neutral votes are required to pass? (multiple answers accepted)
+    - 11 answered: >75% approval to pass (there must be at least 3 times more approval than neutral)
+    - 4 answered: >50% approval to pass (there must be more approval than neutral)
+    - 1 answered: 100% approval to pass (there are no neutral votes, only vetos)
+- When voting for changes on who is or isn't part of the CC network, should we have veto rights, and if so how should they work?
+    - 17 answered: Any voter can veto, which can then be overruled by the majority if the concerns aren't believed to be deserved
+    - Nobody answered: There are no veto rights.
+    - Nobody answered: Any voter can veto, which will immediately cancel the vote
+
+# Prior art
+
+Quilt Community Collab is an indirect replacement of the Modding Interpol, a now extinct private server used to share threat info between communities, with a wider range of activities, such as events and related projects.
+
+# Unresolved questions
+
+The following questions will have to be solved at a later date:
+
+- We aren’t defining any role for administrators of the communities part of the program
+- We only allow for full communities to join, we should look at whenever we also want to invite individuals
+- We are locking Managers to be part of the Quilt Community Team, but we should look at whether we want to allow for other communities to have Managers
```

Comment:
You addressed this in a resolved comment, but I don't think you updated the document

Reply:
I do believe that's something we may want to look at in the future. There would be some great advantages to not having CC 100% controlled by Quilt people. That being said, I'd rather wait a fair bit before looking at this.